modules/cudacodec/include/opencv2/cudacodec.hpp

```diff
@@ -152,14 +152,14 @@ class CV_EXPORTS EncoderCallBack
     @param frameNumber
     @param picType Specify frame type (I-Frame, P-Frame or B-Frame).
      */
-    virtual void onBeginFrame(int frameNumber, PicType picType) = 0;
+    virtual void onBeginFrame(int frameNumber, EncoderCallBack::PicType picType) = 0;
 
     /** @brief Callback function signals that the encoding operation on the frame has finished.
 
     @param frameNumber
     @param picType Specify frame type (I-Frame, P-Frame or B-Frame).
      */
-    virtual void onEndFrame(int frameNumber, PicType picType) = 0;
+    virtual void onEndFrame(int frameNumber, EncoderCallBack::PicType picType) = 0;
 };
```

Comment:
Not wrapped due to enum-related warnings, but no harm in keeping `EncoderCallBack::` there.